### PR TITLE
[DUOS-1853][risk=no] update read-only vote button styles

### PR DESCRIPTION
**Pull request by shaemarks**
[Link to Jira ticket](https://broadworkbench.atlassian.net/browse/DUOS-1853)
Summary of changes:
- Updated logic for `VoteSubsectionHeading` within `CollectionSubmitVoteBox.js` so that read-only scenarios show a sentence detailing the selected vote rather than the usual "My Vote*" heading 
- Hide voting buttons for read-only scenarios 
- Update cypress tests

Notes:
- the admin page view (accessed by clicking linked dar codes in the DarCollectionTable) always uses text instead of vote buttons based on the chairperson / final votes 
- the read-only view for DACs (accessed by clicking linked dar codes in the DarCollectionTable when elections are closed) always uses text based on their votes, which can be YES / NO / NOT SELECTED
- when a chairperson submits a vote on a bucket with datasets (not RP), the vote buttons disappear and text appears reflecting their vote since it cannot be changed once submitted. This closes the election, so the member tab will also have their buttons replaced with text for this bucket since voting is disabled 

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/collection_vote_box/CollectionSubmitVoteBox.js [hunk 1]
```diff
@@ -48,7 +71,7 @@ export default function CollectionSubmitVoteBox(props) {
   const {question, votes, isFinal, isApprovalDisabled, isLoading, adminPage} = props;
 
   useEffect(() => {
-    setIsVotingDisabled(props.isDisabled || (isFinal && submitted) || isLoading);
+    setIsVotingDisabled( props.isDisabled || (isFinal && submitted) || adminPage);
   }, [props.isDisabled, isFinal, submitted, isLoading]);
```
Comment (JVThomas) [inline src/components/collection_vote_box/CollectionSubmitVoteBox.js]: Nit: Add `adminPage` to the dependency array. Even if the value doesn't change after component load, adminPage is a prop and the `useEffect` depends on its value.

**Review by JVThomas — APPROVED** [review]
One nitpick, but other than that the code looks good

**Review by rushtong — APPROVED** [review]
👍🏽

**Review by rushtong — APPROVED** [review]
Still 👍🏽

**Review by JVThomas — APPROVED** [review]
Looks good to me

**Pull request merged**